diff --git a/node_modules/@acid-info/docusaurus-og/lib/server/blog.plugin.js b/node_modules/@acid-info/docusaurus-og/lib/server/blog.plugin.js
index 56d7746..e1b6bc4 100644
--- a/node_modules/@acid-info/docusaurus-og/lib/server/blog.plugin.js
+++ b/node_modules/@acid-info/docusaurus-og/lib/server/blog.plugin.js
@@ -53,7 +53,7 @@ class BlogPlugin {
     loadInstance = async (plugin) => {
         const content = plugin.content;
         const options = plugin.options;
-        content.blogListPaginated.forEach((value) => {
+        (content.blogListPaginated || []).forEach((value) => {
             this.pages.push({
                 data: value,
                 plugin: options,
